### ISSUE TITLE
[Bug] click markline will cause the first column to be selected

### ISSUE THREAD

**Issue by Justin-ZS** [apache/echarts#20609]
### Version

5.5.1

### Link to Minimal Reproduction

[Reproduction](https://echarts.apache.org/examples/en/editor.html?code=PYBwLglsB2AEC8sDeAoWszGAG0iAXMgL4A0as2ApgOaXQAmhSp5AHgIKsQDOT569AIZhBhANoAiALIwJJWBIAqAV0pyFAdUr11SgBbLdAMQBOEXQGVhl5dAkBdci3QBPTjybPY3Smcq9YMVR0dGhBAFtKQgkrKjl-DBcQKIUAI0ETeJDYIRFxAFZ5ACYABnkAZgA2eQBGMtg64vqAFnsybJ8qAGMwbRl6FIluCGhqOPaQzsoevmz0CF7wizAXKlm59FTgEwGTDQh6MD1CIonsogSvdHCMgGsAGRGU4Lnc0UCXjbcuAJrChPQREc5ycjiIQA&_source=echarts-doc-preview)

### Steps to Reproduce

click the markline, you can see the first column is selected.

### Current Behavior

click the markline, you can see the first column is selected.

### Expected Behavior

click the markline, no columns are selected

### Environment

```markdown
- OS:
- Browser:
- Framework:
```

**Comment by helgasoft** [comment]
yes, it seems _markLine_ click does not behave properly with _select_ enabled.
Workaround is to preprocess data and add extra markLines - [Demo](https://echarts.apache.org/examples/en/editor.html?code=CYQwLiC8AEDaCsAaaAmADMgzANgLoG5oAoAWwBthJYBvaATwEEAPASwGcAuaARnmgF8CRUBAB0JEAAcAFMGDRIAPmi1o0APTroIOdGABXEiTrQJAJwDWZFgDsApmyJrywUfptsAFiwBmYaTRsdCQARgD2ZBwA5DZh9lHIAMZhYWbAHLAYaLj8iNRBoRHRsfFJKWkZWTm4AJT4RPw1RGGSYCxxCipO0GApZG2SXNS53WR2AOZ2NukqI2pMzOxD3WoiIFywAEQAsnGbyJsAKvp2-9CbAOp2wJu43XP0i5yziN1sdmYsDhvUK9A2IBIdi4mwAyiAxvs_mA6JJgecQiAzFC1KtwCBXqjoO8xokwNddsB4Zs2LZxpDMaicXY8cssWoWPiSKCYWM6fS1OE0h8LixgGBPFwUJSsfw_g9nEiLAAZWzw1RrLguAT3O78IA).
